<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elaboracion de saldos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>701</width>
      <height>203</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>701</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_output">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_input">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_out">
       <property name="text">
        <string>Directorio de destino</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_proc">
       <property name="text">
        <string>Procesar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="_status">
       <property name="minimumSize">
        <size>
         <width>699</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="combo_version"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_in">
       <property name="text">
        <string>Archivo de origen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_out"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_salir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_in"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Men√∫</string>
    </property>
    <addaction name="actionGuardar_config"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar_config">
   <property name="text">
    <string>Guardar configuracion...</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
